src/controllers/dynakube/activegate/internal/authtoken/reconciler.go

```diff
@@ -17,7 +17,10 @@ import (
 
 const (
 	ActiveGateAuthTokenName   = "auth-token"
-	AuthTokenRotationInterval = time.Hour * 24 * 30
+	AuthTokenRotationInterval = time.Hour*24*30 - AuthTokenBuffer
+
+	// Buffer to avoid warnings in the UI
+	AuthTokenBuffer = time.Hour * 24
```

Comment:
```suggestion

	// Buffer to avoid warnings in the UI
	AuthTokenBuffer = time.Hour * 24
	AuthTokenRotationInterval = time.Hour*24*30 - AuthTokenBuffer
```